src/components/Lesson.js

```diff
@@ -68,7 +53,7 @@ const Lesson = ({ route, navigation, dispatch, completedGrammars }) => {
       onPressHandler={() => completeBtn()}
     >
       <BodyText>{grammar.description}</BodyText>
-      {showCommonness()}
+      <Commonness commonness={grammar.commonness} />
```

Comment:
It's a small nit pick, but `Commonness commonness` is chotto hen.
What do you think about `<Commonness rating={grammar.commonness} />` ?